data/examples.toml

```
@@ -145,6 +145,11 @@ fn = "math"
 source = "math.sil"
 title = "Math"
 
+[[packages]]
+fn = "math-formulas"
```

```suggestion
fn = "math-formulas"
devel = true
```

I don't know if you need this change, but if you even need an example to build with the current default development branch (currently *master*) then you can add this flag.

```
$ nix-shell --run 'sile --version'
SILE v0.15.7 (LuaJIT 2.1.1713773202) [Rust]
```

Ergo if you need anything in v0.15.8 or later, add this flag. Otherwise it will build the example with v0.15.7.